src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -18,26 +21,24 @@ export const plugin: ComputationPlugin = {
 // Include available methods in this array.
 const ABUNDANCE_METHODS = ['median', 'q3', 'variance', 'max'];
 
-// Include known collection variables in this array.
-const ABUNDANCE_COLLECTION_VARIABLES = [
-  { entityId: 'EUPATH_0000808', variableId: 'EUPATH_0009253' },
-];
-
 function variableDescriptorToString(
   variableDescriptor: VariableDescriptor
 ): string {
   return JSON.stringify(variableDescriptor);
 }
 
 export function AbundanceConfiguration(props: ComputationConfigProps) {
-  const [name, setName] = useState('New ranked abundance app');
-  const [collectionVariable, setCollectionVariable] = useState(
-    variableDescriptorToString(ABUNDANCE_COLLECTION_VARIABLES[0])
-  );
+  const [name, setName] = useState('New ranked abundance tool');
   const [rankingMethod, setRankingMethod] = useState(ABUNDANCE_METHODS[0]);
   const { computationAppOverview, addNewComputation } = props;
-  const study = useStudyMetadata();
-  const entities = useStudyEntities(study.rootEntity);
+  const studyMetadata = useStudyMetadata();
+  const entities = useStudyEntities(studyMetadata.rootEntity);
+  // Include known collection variables in this array.
+  const collections = findCollections(studyMetadata.rootEntity).flat();
```

Comment:
Because `findCollections` could be computationally expensive, I would memoize this with a `useMemo`, and also put this memoization logic into a reusable [custom hook](https://reactjs.org/docs/hooks-custom.html):

```typescript
function useFlattenedCollectionVariables(entity: StudyEntity) {
  return useMemo(
    () => findCollections(entity).flat(),
    [entity]
  );
}
```